admin-dev/themes/new-theme/scss/pages/_translation_page.scss

```diff
@@ -1,5 +1,162 @@
+$color-error: #c05c67 !default;
+$color-active: #25b9d7 !default;
+
 .export-translations {
   .export-translations-child {
     padding-left: 50px;
   }
 }
+
+.breadcrumb {
+  small {
+    color: $gray-medium;
+  }
+}
+
+.total-remaining-translations {
+  color: $color-error;
+}
+
+.search-translation {
+  margin-bottom: 1.25rem;
+
+  .alerts {
+    margin-top: 1rem;
+  }
+
+  .col-offset {
+    margin-left: 1rem;
+  }
+
+  .hide {
+    display: none;
+  }
+
+  .summary {
+    font-size: 0.875rem;
+    color: #363a41;
+  }
+
+  .search {
+    width: 100%;
+    margin-top: 1px;
+  }
+
+  .search-button {
+    margin-right: 0.625rem;
+  }
+
+  .alert {
+    float: left;
+    width: 42.63rem;
+    height: 3.438rem;
+    margin-top: 1.875rem;
+
+    i,
+    span {
+      float: left;
+    }
+
+    span {
+      margin-left: 2.5rem;
+    }
+  }
+}
+
+.translation-domains {
+  float: left;
+  min-height: 50rem;
+  background-color: #fff;
+
+  h1 {
+    width: auto;
+    height: 1.375rem;
+    margin-top: 0.9375rem;
+    object-fit: contain;
```

Comment:
Do we support IE11? 

I think yes for the moment, I would like to kick this support but `object-fit` is not supported by IE11 at all, wdyt @PierreRambaud ?

Reply:
We have to support the same browsers as our bootstrap version.